katas/content/getting_started/index.md

```diff
@@ -0,0 +1,70 @@
+# Getting Started
+
+@[section]({
+    "id": "getting_started_welcome",
+    "title": "Welcome to Quantum Katas"
+})
+
+kaˑ ta | kah-tuh
+_The japanese word for "form", a pattern of learning and practicing new skills._
+
+<style>
+.transparent {
+  background: rgba(0, 0, 0, 0.0) !important;
+}
+</style>
+<table>
+    <tbody>
+        <tr class="transparent" style="border:none;">
+            <td rowspan="2" style="border:none;">
+                @[svg]({"path": "./media/Copilot.svg"})
```

Comment:
Being that each image is used once and you're writing inline HTML anyway, did you need a new tag and processing logic for it? You could have just inlined it - i.e. I believe the below would be valid in markdown.

```html
  <td>
    <svg width='100' height='100'>
	  <circle cx=20 cy=20 r=10 fill='blue'>
<!-- etc.. -->
    </svg>
  </td>
```

Done now I guess, so doesn't matter.

Reply:
Yes, that could have been an option. Lots of opportunities to simplify composition of katas.

Reply:
Well, having the macro makes the markdown more readable than embedding the whole SVG.